code/style.css

```diff
@@ -1,7 +1,384 @@
+  height: 400px;
+  overflow: hidden;
+}
+
+
+.header-content {
+  position: relative;
+}
+
+header h2 {
+  color: white;
+}
+
+h1 {
+  font-size: 50px;
+  text-align: center;
+  margin: 0px;
+}
+
+h2 {
+  font-size: 25px;
+  text-align: center;
+  color: lightsalmon;
+}
+
+p {
+  margin: 0;
+}
+
+/*** MAIN ***/
+
+main {
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  align-items: center;
+  justify-content: center;
+  
+}
+
+/*** INTRO ***/
+
+.intro {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  grid-row: span 1;
+  text-align: center;
+  box-sizing: border-box;
+  padding: 20px;
+
+}
+
+/*** SIGN UP ***/
+
+.sign-up {
+  width: 100%;
+  background-color: lightsalmon;
+  color: white;
+  grid-row: span 2;
+  display: grid;
+  grid-template-rows: 80px 2fr;
+  align-items: center;
+}
+
+.sign-up h2 {
+  color: white;
+}
+
+form {
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 15px;
+  justify-content: center;
+  align-items: center;
+  margin: 10px 50px;
+}
+
+form p {
+  font-weight: 500;
+}
+
+input {
+  height: 40px;
+  width: 100%;
+  border: none;
+  background: rgba(255, 255, 255, 0.1);
+  color: white;
+  border-radius: 10px;
+  box-sizing: border-box;
+  padding: 10px;
+}
+
+input:focus {
+  border: 1px solid white;
+  background-color: rgba(255, 255, 255, 0.5)
+}
+
+label {
+  font-weight: 500;
+}
+
+/* Form radio buttons */
+
+form div {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  justify-content: center;
+  align-items: center;
+}
+
+.radio-container {
+  display: block;
+  position: relative;
+  padding-right: 35px;
+  padding-top: 3px;
+  margin-bottom: 12px;
+  cursor: pointer;
+  font-size: 12px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  text-align: right;
+}
+
+.radio-container input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.custom-radio {
+  position: absolute;
+  top: 0;
+  right: 0;
+  height: 24px;
+  width: 24px;
+  background-color: lightsalmon;
+  border-radius: 50%;
+  border: 1px solid white;
+}
+
+.radio-container:hover input ~ .custom-radio {
+  background-color: rgba(255, 255, 255, 0.5);
+}
+
+.radio-container input:checked ~ .custom-radio {
+  background-color: rgba(255, 255, 255, 1);
+}
+
+.custom-radio:after {
+  content:"";
+  position: absolute;
+  display: none;
+}
+
+.radio-container input:checked ~.custom-radio:after {
+  display: block;
+}
+
+.radio-container .custom-radio:after {
+  top: 6px;
+  left: 6px;
+  width: 12px;
+  height: 12px;
+  border-radius: 50%;
+  background-color: lightsalmon;
+}
+
+
+button {
+  margin: 20px 0;
+  height: 40px;
+  border: none;
+  background: rgba(255, 255, 255, 1);
+  color: lightsalmon;
+  border-radius: 10px;
+  font-weight: bold;
+  font-size: inherit;
+  font-family: inherit;
+}
+
+/*** FAQ ***/
+
+.accordion {
+  width: 100%;
+  height: 300px;
```

Comment:
Regarding when to use and not use fixed widths/heights. Here's a good example, when one of your .question-containers are active and the answer is shown it expands outside of the .accordion-containers fixed height. 
<img width="471" alt="Screenshot 2020-09-10 at 15 29 32" src="https://user-images.githubusercontent.com/14332326/92737261-45745400-f37b-11ea-8222-9fd043e3bdab.png">

This only happens in mobile view however. So don't use fixed heights or widths if the size of the containers content might change. An option here would be to use `min-height: 300px;` then the container will never be less than 300px in height, but can expand if the content of it does. This is tricky and the beginning but since you asked about it I thought I should give you a thorough answer.